run-go-critic.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+if command -v gocritic &> /dev/null ; then
+    echo "gocyclo not installed or available in the PATH" >&2
```

Comment:
Looks like a copy/paste error here:

```suggestion
    echo "gocritic not installed or available in the PATH" >&2
```